<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListFlags</class>
 <widget class="QDialog" name="ListFlags">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>List and Remove Flags</string>
  </property>
  <widget class="QListWidget" name="lw_Flags">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>711</width>
     <height>401</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Remove">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cancel">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>70</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_SelectAll">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>340</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_SelectNone">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>380</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Select None</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>ListFlags</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>795</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
